c2/c2-protocol/c2-protocol-api/src/main/java/org/apache/nifi/c2/protocol/api/ProcessorBulletin.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.c2.protocol.api;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.io.Serializable;
+import java.util.Date;
+
+public class ProcessorBulletin  implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private Date timestamp;
+    private long id;
+    private String nodeAddress;
+    private String level;
+    private String category;
+    private String message;
+    private String groupId;
+    private String groupName;
+    private String groupPath;
+    private String sourceId;
+    private String sourceName;
+    private String flowFileUuid;
+
+    @Schema(description = "When this bulletin was generated.")
```

Comment:
It seems the convention in the C2 API is to not have dots at the end of the descriptions. Could you remove these?

Reply:
Done